### ISSUE TITLE
depracated SubfieldBase

### ISSUE THREAD

**Issue by banuni** [fle/django-multi-email-field#3]
raises a depracation warning:

```
multi_email_field/fields.py:8: RemovedInDjango110Warning: SubfieldBase has been deprecated. Use Field.from_db_value instead.
  class MultiEmailField(models.Field):
```

**Comment by aapris** [comment]
Vote this up. Fix is needed before Django can be updated to 1.10.

**Comment by aapris** [comment]
Actually it looks like this is fixed in #4

**Comment by fle** [comment]
Thank you!
